test/model/channel_test.dart

```diff
@@ -68,6 +68,22 @@ void main() {
       ));
       checkUnified(store);
     });
+
+    test('removed then added by events', () async {
```

Comment:
```suggestion
    test('unsubscribed then subscribed by events', () async {
```

The group is about "stream/sub data" in general, so this otherwise sounds like it's probably about removing the whole stream.

Reply:
Separately, since this is a regression test for a particular bug that we actually had, let's link that context:
```suggestion
    test('removed then added by events', () async {
      // Regression test for: https://chat.zulip.org/#narrow/channel/48-mobile/topic/Unsubscribe.20then.20resubscribe.20to.20channel/with/2160241
```